docs/helm/values/local-dev-values.yaml

```diff
@@ -0,0 +1,121 @@
+externalPort: "80"
+externalProtocol: http
+externalHost: localhost
+activemq:
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "512Mi"
+    limits:
+      cpu: "1000m"
+      memory: "1Gi"
+repository:
+  replicaCount: 1
+  licenseSecret: acslicense
```

Comment:
Please remove that as the corresponding volume secret, do not seem to be optional